instant-xml-macros/src/common.rs

```diff
@@ -0,0 +1,43 @@
+use syn::{DeriveInput, Lit, Meta, NestedMeta};
+
+pub fn retrieve_default_namespace(input: &DeriveInput) -> Option<String> {
+    for item in get_meta_items(&input.attrs) {
```

Comment:
Try to reduce rightward drift by using early `return` or `continue`. Instead of unwrapping `Option`s we can just use `?` on them to return `None` to the caller.

Reply:
I rewritten this function, looks much better now, please take a look.